test/e2e/tests/phishing-controller/mocks.js

```diff
@@ -10,7 +10,9 @@ const {
 const lastUpdated = 1;
 const defaultHotlist = { data: [] };
 const defaultC2DomainBlocklist = {
-  recentlyAdded: [],
+  recentlyAdded: [
+    '33c8e026e76cea2df82322428554c932961cd80080fa379454350d7f13371f36',
```

Comment:
what is this value for?

Reply:
This value is the hash of `malicious.localhost` I added a comment for context.